.gitignore

```diff
@@ -15,4 +15,6 @@ working*
 
 App.config.mine
 src/packages/*
-!src/packages/repositories.config
\ No newline at end of file
+!src/packages/repositories.config
+src/.vs/config/applicationhost.config
```

Comment:
not sure why this is here? I am pretty sure, if anything, this should read .vs/ to ignore that entire folder